test/Microsoft.TestPlatform.CommunicationUtilities.UnitTests/TestRequestSenderTests.cs

```diff
@@ -277,7 +308,7 @@ public void StartTestRunWithSourcesShouldCallHandleTestRunStatsChange()
 
             waitHandle.WaitOne();
 
-            this.mockCommunicationManager.Verify(mc => mc.SendMessage(MessageType.StartTestExecutionWithSources, runCriteria), Times.Once);
+            this.mockCommunicationManager.Verify(mc => mc.SendMessage(MessageType.StartTestExecutionWithSources, runCriteria, this.version), Times.Once);
```

Comment:
Better to extract a method `VerifySendMessage(messageType, payload)` for all repetitions.